man/systemd.network.xml

```diff
@@ -4000,6 +4000,18 @@ ServerAddress=192.168.0.1/24</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ReachableTimeSec=</varname></term>
+
+        <listitem>
+          <para>Configures the reachable time, used by by the Neighbor Unreachability Detection algorithm
+          that clients assume a neighbor is reachable after having received a reachability confirmation.
+          Takes a timespan in the range 0…4294967295 msec. Defaults to 0.</para>
```

Comment:
"Configures the time, used in the Neighbor Unreachability Detection algorithm, for which clients can assume a neighbor is reachable after having received a reachability confirmation. Time a time span in the range  0…4294967295 ms. Defaults to 0."

(Please use "ms". "ms" is the SI unit.)

What does it mean when 0 is used? The code below ignores that value, so maybe this should say "When set to 0, this option is not sent to clients. Defaults to 0.".

Reply:
Thanks. Applied the suggested sentence.

The value is used in RA header. So, 0 is also a valid value, and will be sent to clients as is. When a client receives RA with 0 reachable time, then the value will be handled as unspecified and will be ignored.